packages/go_router/lib/src/state.dart

```diff
@@ -4,13 +4,13 @@
 
 import 'package:flutter/foundation.dart';
 
-import 'go_route_information_parser.dart';
+import 'configuration.dart';
 
 /// The route state during routing.
 class GoRouterState {
   /// Default constructor for creating route state during routing.
   GoRouterState(
-    this._delegate, {
+    this._configuration, {
```

Comment:
GoRouterState is somewhat misnamed... It's not really the "state" because it's not stored anywhere - it's constructed by the builder to provide the user with information. So I don't expect users to be calling the constructor. Maybe we should add a TODO to make it abstract so we can keep the constructor private...